lib/incito-browser/incito.js

```diff
@@ -1,308 +1,614 @@
+            contents = text.replace(/\n/g, '<br>');
 
-                triggeredVisibleRendered = true;
+            if (
+                Array.isArray(view.font_family) &&
+                view.font_family.length > 0
+            ) {
+                styles['font-family'] = view.font_family.join(', ');
             }
 
-            if (this.renderedOutsideOfViewport) {
-                this.lazyload(0);
+            if (view.text_size != null) {
+                styles['font-size'] = `${view.text_size}px`;
             }
-        };
 
-        this.el.className = 'incito';
-        if (this.incito.locale != null) {
-            this.el.setAttribute('lang', this.incito.locale);
-        }
+            if (view.line_spacing_multiplier != null) {
+                styles['line-height'] = view.line_spacing_multiplier;
+            }
 
-        loadFonts(this.incito.font_assets);
-        this.applyTheme(this.incito.theme);
+            if (view.text_color != null) {
+                styles.color = view.text_color;
+            }
 
-        this.containerEl.appendChild(this.el);
+            if (textStyles.indexOf('bold') !== -1) {
+                styles['font-weight'] = 'bold';
+            }
 
-        // do first render synchronously unless we're very lazy
-        if (this.renderLaziness === 2) {
-            this.renderCallbackHandle = requestIdleCallback(render);
-        } else {
-            syncIdleCallback(render);
-        }
+            if (textStyles.indexOf('italic') !== -1) {
+                styles['font-style'] = 'italic';
+            }
 
-        document.addEventListener('scroll', this.lazyloader, true);
-        window.addEventListener('resize', this.lazyloader, false);
+            if (Array.isArray(view.text_decoration_line)) {
+                styles['text-decoration-line'] = view.text_decoration_line.join(
+                    ' '
+                );
+            }
 
-        return this;
-    }
+            const textShadow = getTextShadow(view);
 
-    destroy() {
-        cancelIdleCallback(this.renderCallbackHandle);
-        this.containerEl.removeChild(this.el);
+            if (isDefinedStr(view.text_shadow)) {
+                styles['text-shadow'] = view.text_shadow;
+            } else if (textShadow != null) {
+                styles[
+                    'text-shadow'
+                ] = `${textShadow.dx}px ${textShadow.dy}px ${textShadow.radius}px ${textShadow.color}`;
+            }
 
-        document.removeEventListener('scroll', this.lazyloader, true);
-        window.removeEventListener('resize', this.lazyloader, false);
+            if (view.text_alignment === 'left') {
+                styles['text-align'] = 'left';
+            } else if (view.text_alignment === 'center') {
+                styles['text-align'] = 'center';
+            } else if (view.text_alignment === 'right') {
+                styles['text-align'] = 'right';
+            }
 
-        this.trigger('destroyed');
```

Comment:
The Controls of IncitoPublicationKit relies on this `destroyed` event to remove event handlers: https://github.com/shopgun/shopgun-js-sdk/blob/develop/lib/kits/incito-publication/controls.js#L13